src/ceph-disk/ceph_disk/main.py

```diff
@@ -3494,7 +3495,7 @@ def mount_activate(
 
     # TODO always using mount options from cluster=ceph for
     # now; see http://tracker.newdream.net/issues/3253
-    mount_options = get_mount_options(cluster='ceph', fs_type=fstype)
+    mount_options = get_mount_options(cluster=cluster, fs_type=fstype)
```

Comment:
i guess we can remove the "TODO" comment above with this change =)

Reply:
also could you add "Fixes: http://tracker.ceph.com/issues/3253" in the commit message

Reply:
done